deploy/crds/neutron.openstack.org_ovsnodeosps_crd.yaml

```diff
@@ -34,9 +34,6 @@ spec:
             nic:
               description: NIC for ovn encap ip
               type: string
-            ovnSbRemote:
-              description: ovn south db ip
-              type: string
```

Comment:
Should we add the name of the ovn-connection configmap to the CRD? Don't do this because I'm not sure, and this is fine for now.